### DISCUSSION TITLE
Question: Wotlk 3.3.5a client on M1 chip?

### DISCUSSION THREAD

**Discussion by lakarpusky** [The-Wineskin-Project/WineskinServer#335] [General]
Is there an update on how to get it running for M1 chips? 

https://www.mangosrumors.org/how-to-run-wow-32-bit-on-mac-os/
It does opens but crashes in the chars room:
<img width="336" alt="Screenshot 2022-12-01 at 11 38 42 PM" src="https://user-images.githubusercontent.com/25494196/208267081-5d049f42-233c-4ee2-9511-32dd367b4594.png">

**Comment by Faetu** [comment]
Any news?

**Comment by Gcenx** [comment]
The 32Bit WoW client don’t work in CX22.1.1-10 or lower, not tried CrossOver-23 beta builds but it’s lightly not changes.

**Comment by pcristofol** [comment]
Just tried the current Turtle WoW client with WS11WineCX64Bit23.0.1-wip and WS11WineCX64Bit22.1.1-14_D3DMetal-v1.0.4 on my Air M1 ... same error :(


```
This application has encountered a critical error:
ERROR #132 (0x85100084) Fatal Exception
Program:
C: Program Files
dient 1165 WoWFoV.exe
Exception:
0C000001D
(ILLEGAL_INSTRUCTION) at 0107:006FA876
```

**Reply by Gcenx** [reply]
As I’ve said already in this thread 32Bit Wow doesn’t work, I’ve not tested with CX23.x
